tox.ini

```diff
@@ -1,388 +1,58 @@
-           /bin/bash -c '{envpython} -m pytest --durations 0 -v {posargs} -n4 --profile=snowflake test/rpc'
-deps =
-    ./core
-    ./plugins/snowflake
-    -r ./dev_requirements.txt
-
-[testenv:integration-bigquery-py37]
-basepython = python3.7
-passenv = *
-setenv =
-    HOME=/home/tox
-commands = /bin/bash -c '{envpython} -m pytest --durations 0 -v -m profile_bigquery {posargs} -n4 test/integration/*'
-deps =
-    ./core
-    ./plugins/bigquery
-    -r ./dev_requirements.txt
-
-[testenv:integration-redshift-py37]
-basepython = python3.7
-passenv = *
-setenv =
-    HOME=/home/tox
-commands = /bin/bash -c '{envpython} -m pytest --durations 0 -v -m profile_redshift {posargs} -n4 test/integration/*'
-deps =
-    ./core
-    ./plugins/postgres
-    ./plugins/redshift
-    -r ./dev_requirements.txt
-
-[testenv:integration-presto-py37]
-basepython = python3.7
-passenv = *
-setenv =
-    HOME=/home/tox
-commands = /bin/bash -c '{envpython} -m pytest --durations 0 -v -m profile_presto {posargs} -n4 test/integration/*'
-deps =
-    ./core
-    ./plugins/presto
-    -r ./dev_requirements.txt
-
-
-[testenv:explicit-py37]
-basepython = python3.7
-passenv = *
-setenv =
-    HOME=/home/tox
-commands = /bin/bash -c '{envpython} -m pytest --durations 0 -v {posargs}'
-deps =
-    -r ./editable_requirements.txt
-    -r ./dev_requirements.txt
-
-[testenv:unit-py38]
-basepython = python3.8
-commands = /bin/bash -c '{envpython} -m pytest --durations 0 -v {posargs} -n4 test/unit'
-deps =
-    -r ./requirements.txt
-    -r ./dev_requirements.txt
-
-
-[testenv:integration-postgres-py38]
-basepython = python3.8
-passenv = *
-setenv =
-    HOME=/home/tox
-commands = /bin/bash -c '{envpython} -m pytest --durations 0 -v -m profile_postgres {posargs} -n4 test/integration/*'
-           /bin/bash -c '{envpython} -m pytest --durations 0 -v {posargs} -n4 --profile=postgres test/rpc'
-deps =
-    ./core
-    ./plugins/postgres
-    -r ./dev_requirements.txt
-
-[testenv:integration-snowflake-py38]
-basepython = python3.8
-passenv = *
-setenv =
-    HOME=/home/tox
-commands = /bin/bash -c '{envpython} -m pytest --durations 0 -v -m profile_snowflake {posargs} -n4 test/integration/*'
-           /bin/bash -c '{envpython} -m pytest --durations 0 -v {posargs} -n4 --profile=snowflake test/rpc'
-deps =
-    ./core
-    ./plugins/snowflake
-    -r ./dev_requirements.txt
-
-[testenv:integration-bigquery-py38]
-basepython = python3.8
-passenv = *
-setenv =
-    HOME=/home/tox
-commands = /bin/bash -c '{envpython} -m pytest --durations 0 -v -m profile_bigquery {posargs} -n4 test/integration/*'
-deps =
-    ./core
-    ./plugins/bigquery
-    -r ./dev_requirements.txt
-
-[testenv:integration-redshift-py38]
+[testenv:flake8]
 basepython = python3.8
-passenv = *
-setenv =
-    HOME=/home/tox
-commands = /bin/bash -c '{envpython} -m pytest --durations 0 -v -m profile_redshift {posargs} -n4 test/integration/*'
+skip_install = true
+commands = flake8 --select=E,W,F --ignore=W504,E741 \
```

Comment:
Can we change flake8 to allow longer lines here?

Reply:
ya, just set it to 99 !